game/hud/src/widgets/Welcome/index.tsx

```diff
@@ -0,0 +1,130 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import {client, events, faction, restAPI} from 'camelot-unchained';
+import * as React from 'react';
+
+export interface WelcomeProps {
+};
+
+export interface WelcomeState {
+  visible: boolean;
+  message: JSX.Element[];
+};
+
+export interface WelcomeData {
+  id: string;
+  message: string;
+  duration: number;
+}
+
+declare const cuAPI: any;
+
+class Welcome extends React.Component<WelcomeProps, WelcomeState> {
+  public name: string = 'Welcome';
+
+  constructor(props: WelcomeProps) {
+    super(props);
+
+    let visibilityDefault: boolean = true;
+
+    try {
+      const delayInMin: number = 24 * 60;
+      // const delayInMin: number = 1; // 1 min for testing
+      const savedDelay = localStorage.getItem('cse-welcome-hide-start');
+      const currentDate: Date = new Date();
+      const savedDelayDate: Date = new Date(JSON.parse(savedDelay));
+      savedDelayDate.setTime(savedDelayDate.getTime() + (delayInMin*60*1000));
+      if (currentDate < savedDelayDate) visibilityDefault = false;
+    } catch (error) {
+      console.log(error);
+    }
+
+    //**TODO: set a decent default message. API is sometimes slow to return.
```

Comment:
This should say something like "Welcome to Camelot Unchained! Loading welcome message..." or something like that then if you could add that to this in place of the default.